docs/code-examples/bar_chart.rs

```diff
@@ -1,13 +1,13 @@
 //! Create and log a bar chart
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let (rec, storage) = rerun::RecordingStreamBuilder::new("rerun_example_bar_chart").memory()?;
+    let rec = rerun::RecordingStreamBuilder::new("rerun_example_bar_chart")
+        .spawn(&rerun::SpawnOptions::default(), None)?;
```

Comment:
Maybe we can have a `spawn_default()` or something? This is a lot of noise for our users

Reply:
Or `.spawn()` and `.spawn_opt(…)`, which seems fairly common in Rust

Reply:
`spawn_opt` sounds good